docs/DeveloperGuide.md

```diff
@@ -97,17 +105,20 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 
 ![Structure of the Model Component](images/ModelClassDiagram.png)
 
-**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
+**API** : [`Model.java`](https://github.com/AY2021S1-CS2103-W14-4/tp/blob/master/src/main/java/seedu/clinic/model/Model.java)
 
 The `Model`,
 
 * stores a `UserPref` object that represents the user’s preferences.
-* stores the address book data.
+* stores the clinic data.
 * exposes an unmodifiable `ObservableList<Supplier>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
 * does not depend on any of the other three components.
 
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Supplier` references. This allows `AddressBook` to only require one `Tag` object per unique `Tag`, instead of each `Supplier` needing their own `Tag` object.<br>
+<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a
+ more OOP) model is given below. It has a `Tag` list in the `Clinic`, which `Supplier` references. This allows
```

Comment:
Just to clarify will this still be relevant for CLI-nic and also, is it still Tag, or would it be Remarks?

Reply:
Okay as per user guide, we only have Tag for products so yes this should be Remarks!

Reply:
It shouldn't be remarks either, it would look like this now and should be pretty straightforward without explanation. We can improve on it at a later time because UML is still rather unstable.
![BetterModelClassDiagram](https://user-images.githubusercontent.com/19368296/96555682-a718ce00-12ea-11eb-85f1-80426e7a4caa.png)

Reply:
I see! Agreed, I think we can improve on this paragraph after we finalise our UML towards the end. Looks good to merge then! 👍